package.json

```diff
@@ -83,40 +81,17 @@
     "eslint-plugin-jsx-a11y": "^6.1.1",
     "eslint-plugin-prettier": "^2.6.2",
     "eslint-plugin-react": "^7.11.1",
-    "husky": "^1.0.0-rc.15",
+    "husky": "^4.2.3",
     "jest": "^23.6.0",
     "prettier": "^1.14.3",
     "react": "^16.5.2",
     "react-dom": "^16.5.2",
     "react-test-renderer": "^16.5.2",
-    "release-script": "^1.0.2",
     "rimraf": "^2.6.1",
-    "semantic-release": "^15.9.16",
-    "semantic-release-alt-publish-dir": "^2.1.1",
     "sinon": "^6.3.4",
     "travis-deploy-once": "^5.0.8",
```

Comment:
Looks like Travis config was removed - do we still need this dependecies.  Looks like it has been deprecated anyway - https://www.npmjs.com/package/travis-deploy-once

Reply:
I see also that the command was removed from the scripts node in package.json so it seems certain that we can remove this.

Reply:
No we shouldn't. I'll remove that as well. Nice catch!

Reply:
This has now been removed.